contracts-periphery/foundry.toml

```diff
@@ -21,7 +21,12 @@
   invariant = { runs = 1000 }
 
 [rpc_endpoints]
+  arbitrum = "${ARBITRUM_RPC_URL}"
```

Comment:
In `.env.example` you call it `ARBITRUM_ONE_RPC_URL`. I believe StdChains would also look for `ARBITRUM_ONE_RPC_URL` so that's probably what we should use

Reply:
great catch!

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/504/commits/ed081a587b4cf48808b00edf7bb85c4b932608ad